### PR TITLE
MVP implementation for Meetups rsvp on site via UM

### PR DESCRIPTION
**Pull request by ric9176**

### CODE REVIEW

File: src/pages/community/meetups/graphql-deep-dive-with-vladimir-novick.js [hunk 1]
```diff
@@ -60,12 +42,12 @@ const graphqlDeepDiveBerlin = () => (
           <Header
             titleLines={['GraphQL deep dive with Vladimir Novick']}
             links={
-              { text: 'Meetup Details', to: '#curriculum' },
-              { text: 'Buy tickets', to: '#target-audience' }
-
+              ({ text: 'Meetup Details', to: '#curriculum' },
+              { text: 'Buy tickets', to: '#target-audience' })
```
Comment (alexlbr) [inline src/pages/community/meetups/graphql-deep-dive-with-vladimir-novick.js]: @ric9176 buy tickets to target audience?

File: src/pages/react/training/workshops/design-system-styling-in-react/index.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ import { LIST_TWO_COL } from 'src/components/curriculum/selectCurriculumLayout'
 
 const StylingDesignSystemWorkshop = () => (
```
Comment (alexlbr) [inline src/pages/react/training/workshops/design-system-styling-in-react/index.js]: @ric9176 StylingDesignSystemWorkshop?

Comment (ric9176) [reply src/pages/react/training/workshops/design-system-styling-in-react/index.js]: I guess bc it was initially copied, I'll go through and update the naming ;-)

Comment (shakersinthedark) [reply src/pages/react/training/workshops/design-system-styling-in-react/index.js]: Yeah, that's why

**Pull request merged**